tests/Integration/Behaviour/Features/Context/CarrierFeatureContext.php

```diff
@@ -132,6 +132,8 @@ public function createCountry($countryName, $isoCode, $zoneName)
         $country->id_zone = $this->zones[$zoneName]->id;
         $country->active = true;
         $country->save();
+
+        $this->getSharedStorage()->set($countryName, (int) $countryId);
```

Comment:
Ok for the modification, however this endpoint should be in a Country context no? Maybe it's not possible because of all those local fields that store the objects but it's probably not the best implementation (can be fixed in another PR though)